src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -205,6 +205,9 @@ public function setConnection(\PDO $connection)
     {
         $this->connection = $connection;
 
+        // ensure connection throws exceptions on errors (SQL syntax errors, etc)
+        $this->connection->setAttribute(\PDO::ATTR_ERRMODE, \PDO::ERRMODE_EXCEPTION);
```

Comment:
Please remove this line, this is done here already https://github.com/cakephp/phinx/pull/1122/files